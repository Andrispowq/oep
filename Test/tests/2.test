mzn
x 0.5
s 0.7
x 1.6
z 17
s 1.34