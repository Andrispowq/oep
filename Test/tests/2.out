Round 0
CO2Layer (Thickness: 0.7749999999999999
Oxygene (Thickness: 1.7850000000000001
OzoneLayer (Thickness: 16.15
CO2Layer (Thickness: 1.58
Oxygene (Thickness: 0.8500000000000001
Round 1
CO2Layer (Thickness: 0.7749999999999999
Oxygene (Thickness: 0.8925000000000001
OzoneLayer (Thickness: 17.042499999999997
CO2Layer (Thickness: 1.58
Round 2
CO2Layer (Thickness: 0.73625
Oxygene (Thickness: 0.886625
OzoneLayer (Thickness: 17.087124999999997
CO2Layer (Thickness: 1.5010000000000001
Round 0
CO2Layer (Thickness: 0.73625
Oxygene (Thickness: 0.75363125
OzoneLayer (Thickness: 16.232768749999998
CO2Layer (Thickness: 1.6339937500000001
Oxygene (Thickness: 0.8543562499999999
Round 1
CO2Layer (Thickness: 0.73625
OzoneLayer (Thickness: 16.609584374999997
CO2Layer (Thickness: 1.6339937500000001